front/src/routes/scene/edit-scene/triggers/GladysStartTrigger.jsx

```diff
@@ -0,0 +1,16 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+
+class SunriseSunsetTrigger extends Component {
+  render({}, {}) {
+    return (
+      <div>
+        <div class="row">
+          <div class="col-sm-12">COUCOU //TODO</div>
```

Comment:
Forgot this? 😄 Maybe let's add a little sentence explaining what this trigger is doing?

![Screenshot 2023-10-16 at 15 55 05](https://github.com/GladysAssistant/Gladys/assets/7365207/6ddc0b5e-c084-4017-a264-a65ad4add4ab)